spec/controllers/general_feedbacks_controller_spec.rb

```diff
@@ -0,0 +1,119 @@
+require "rails_helper"
+
+RSpec.describe GeneralFeedbacksController, type: :controller do
```

Comment:
It would be good to start writing requrest specs and get rid of controller specs that are not supported anymore

Reply:
I will replace this with a request spec, in a separate PR.

Reply:
In https://github.com/DFE-Digital/teaching-vacancies/pull/2774